ods_ci/tests/Tests/1000__model_serving/1010__model_serving_triton_on_modelmesh/1010__model_serving_triton_on_modelmesh.robot

```diff
@@ -0,0 +1,100 @@
+*** Settings ***
+Documentation     Suite of test cases for Triton in Modelmesh
+Library           OperatingSystem
+Library           ../../../../libs/Helpers.py
+Resource          ../../../Resources/Page/ODH/JupyterHub/HighAvailability.robot
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHModelServing.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDashboardSettingsRuntimes.resource
+Resource          ../../../Resources/Page/ODH/Monitoring/Monitoring.resource
+Resource          ../../../Resources/OCP.resource
+Resource          ../../../Resources/CLI/ModelServing/modelmesh.resource
+Resource          ../../../Resources/Common.robot
+Suite Setup       Triton On Kserve Suite Setup
+Suite Teardown    Triton On Kserve Suite Teardown
+Test Tags         Kserve
+
+
+*** Variables ***
+${INFERENCE_REST_INPUT_ONNX_FILE}=    @tests/Resources/Files/triton/kserve-triton-onnx-rest-input.json
+${PRJ_TITLE}=    ms-triton-project-mm
+${PRJ_DESCRIPTION}=    project used for model serving triton runtime tests
+${MODEL_CREATED}=    ${FALSE}
+${ONNX_MODEL_NAME}=    densenet_onnx
+${ONNX_MODEL_LABEL}=     densenetonnx
+${ONNX_RUNTIME_NAME}=    modelmesh-triton
+${RESOURCES_DIRPATH}=        tests/Resources/Files/triton
+${ONNX_MODELMESH_RUNTIME_FILEPATH}=    ${RESOURCES_DIRPATH}/triton_onnx_modelmesh_runtime.yaml
+${EXPECTED_INFERENCE_REST_OUTPUT_FILE}=      tests/Resources/Files/triton/modelmesh-triton-onnx-rest-output.json
```

Comment:
you could use ${RESOURCES_DIRPATH} inside this path

Reply:
Changed